src/ast/ddl.rs

```diff
@@ -208,6 +229,12 @@ pub enum AlterTableOperation {
         old_partitions: Vec<Expr>,
         new_partitions: Vec<Expr>,
     },
+    /// REPLICA IDENTITY { DEFAULT | USING INDEX index_name | FULL | NOTHING }
+    ///
+    /// Note: this is a PostgreSQL-specific operation.
```

Comment:
Can we add a link to the docs ideally so folks can easily find where the syntax comes from?